Ghidra/Debug/Debugger-agent-frida/src/main/java/agent/frida/manager/cmd/AbstractFridaCommand.java

```diff
@@ -117,8 +115,9 @@ public void parse(String result, Object data) {
 			} else {
 				manager.getEventListeners().fire.consoleOutput(object+"\n", 0);		
 			}
-			if (res.equals("[]")) {
+			if (res.toString().equals("[]")) {
```

Comment:
would be better if you flipped the const and non-const around: "[]".equals(res.toString())